wordpress-mobile/WordPress-iOS
# Issue/aztec media options

This PR adds the possibility to edit align and size classes on images inside the Aztec editor.

To test:
 - Create a new post using Aztec
 - Add an image
 - Long press on the image
 - Select Details
 - Change size and alignment
 - Press done
 - Check if properties are applied to image

Needs review: @jleandroperez
